commitlint/helpers.ts

```diff
@@ -22,13 +22,7 @@ export abstract class Helpers {
     ): string {
         if (potentialString === null || potentialString === undefined) {
             // otherwise, String(null) might give us the stupid string "null"
-            throw new Error(
-                "Unexpected " +
-                    paramName +
-                    "===null or " +
-                    paramName +
-                    "===undefined happened"
-            );
+            return "";
```

Comment:
@realmarv this is unacceptable, it should return null, not `""`

Reply:
If I return null here I have to add extra indentation in all rules for `if (xStr !== null)` condition. Is that ok?

Reply:
that is ok; what is not ok is masquerading a missing string with an empty string; they are not the same thing